radio/src/opentx.cpp

```diff
@@ -225,6 +225,13 @@ void per10ms()
 
   outputTelemetryBuffer.per10ms();
 
+//OW
+#if defined(TELEMETRY_MAVLINK)
+  mavlinkTelem.tick10ms();
```

Comment:
We need to find a way to include this into the normal telemetry processing path, if that is possible. And please try to make it conditional on MAVLink serial being activated. Does it make sense?

Reply:
I don't disagree with either, but both IMHO are not so easy ... 
for 1. see my response below to https://github.com/EdgeTX/edgetx/pull/26#discussion_r633081750
for 2. see what I've said in the above

Reply:
Ok, we will review this again once I get into the details.